### PR TITLE
fix: avoid associative array written in client lib registry config

### PR DESCRIPTION
**Pull request by pnal**
The fix for [exactly same issue](https://github.com/oat-sa/extension-tao-itemqti/pull/1628/files#diff-be6ef8f839fc7891cab9ff31c4e02183177bb3345686fa8a06cad738216babb9R79) but for a reverse case when we remove a plugin, not registering.

### CODE REVIEW

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [ ]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful
Before
image

After
image

File: model/QtiCreatorClientConfigRegistry.php [hunk 1]
```diff
@@ -30,7 +30,7 @@
  */
 class QtiCreatorClientConfigRegistry extends ClientLibConfigRegistry
 {
-    const CREATOR = "taoQtiItem/controller/creator/index";
+    public const CREATOR = "taoQtiItem/controller/creator/index";
```
Comment (Karol-Stelmaczonek) [inline model/QtiCreatorClientConfigRegistry.php:33]: suggestion: can we make it private if its used only here?

**Review by Karol-Stelmaczonek — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: model/QtiCreatorClientConfigRegistry.php (hunk 1 above)
Comment (pnal) [reply model/QtiCreatorClientConfigRegistry.php:33]: No, it is used somewhere

**Pull request merged**